tests.py

```diff
@@ -542,17 +559,19 @@ def test_defused_gzip_response(self):
 
 def test_main():
     suite = unittest.TestSuite()
-    suite.addTests(unittest.makeSuite(TestDefusedcElementTree))
-    suite.addTests(unittest.makeSuite(TestDefusedElementTree))
-    suite.addTests(unittest.makeSuite(TestDefusedMinidom))
-    suite.addTests(unittest.makeSuite(TestDefusedMinidomWithParser))
-    suite.addTests(unittest.makeSuite(TestDefusedPulldom))
-    suite.addTests(unittest.makeSuite(TestDefusedSax))
-    suite.addTests(unittest.makeSuite(TestXmlRpc))
-    if lxml is not None:
```

Comment:
the removed check now leads to very cryptic errors in the tests when lxml is not installed.

    TypeError: catching classes that do not inherit from BaseException is not allowed
    AttributeError: 'NoneType' object has no attribute 'fromstring'